handler/handler.go

```diff
@@ -39,7 +42,10 @@ func handle(request types.Request) {
 			Name: proto.String("Eremetic"),
 			User: proto.String(""),
 		},
-		Scheduler: scheduler,
+		Scheduler:        scheduler,
+		BindingAddress:   net.ParseIP("0.0.0.0"),
```

Comment:
I don't know why I have to do this but otherwise it tries to bind() to the publish ip